packages/app/styles/global.css

```diff
@@ -30,3 +30,24 @@ body {
   border-color: #3684c6;
   background-color: #3684c6;
 }
+
+.ant-modal-body {
+  padding: 0px 24px;
+}
+
+.notion-image-inset {
+  width: 100%;
+}
+
+.notion-asset-wrapper {
+  width: 100% !important;
+}
+
+.notion-asset-wrapper > div {
+  padding-bottom: 0px !important;
+}
+
+.notion-list {
+  margin: 0px;
+  padding-inline-start: 20px;
```

Comment:
I know I did this with you but we should align the toggles with the list, so move the toggles to the right a bit or move this to the left

Reply:
fixed